doc/devel/min_dep_policy.rst

```diff
@@ -51,6 +51,13 @@ versions no longer support our minimum NumPy or Python.
 
 We will work around bugs in our dependencies when practical.
 
+IPython and Matplotlib do not technically depend on each other, however there
+is practical coupling between the projects.  There is no dependency to update,
```

Comment:
```suggestion
is practical coupling between the projects via the matplotlib inline backend. 
There is no explicitly written dependency to update,
```
- We should at least hint at the reason for the coupling.
- "Dependency" is a bit vague. It's either the general term for coupling or the codified package relation.
  You want the latter. Is "explicitly written dependency"

Reply:
If you want to mention the reason for the coupling then it is not just via the Matplotlib inline backend, it is use of any Matplotlib backend with either IPython or any of the Jupyter subcommands.

Reply:
Yup, I just wanted to give an idea. I believe most people won't know what "use of any Matplotlib backend with either IPython or any of the Jupyter subcommands." means. Maybe (but still not fully correct) "via the ``%matplotlib`` command magic.

Reply:
I had written a sentence about event loops but decided that was too much detail.

Reply:
New suggestion
```suggestion
is practical coupling for the integration of Matplotlib into Ipython.
```